crmsh/utils.py

```diff
@@ -254,8 +258,8 @@ def multi_input(prompt=''):
             text = input(prompt)
         except EOFError:
             return None
-        err_buf.incr_lineno()
         if options.regression_tests:
+            logger_utils.incr_lineno()
```

Comment:
This changes the original logic a little. Just to bring the attention to double check if it does make sense.

Reply:
Never mind, lineno here is only  used for origin regression tests
If the tests results doesn't complain, then it works:)